openTSNE/tsne.py

```diff
@@ -70,6 +70,10 @@ def _handle_nice_params(optim_params: dict) -> None:
 
     optim_params["n_jobs"] = n_jobs
 
+    # Determine learning rate if requested
+    if optim_params.get("learning_rate", "auto") == "auto":
+        optim_params["learning_rate"] = max(200, embedding.shape[0] / 12)
```

Comment:
Maybe change 12 to the early exaggeration factor here?

Reply:
I wanted to do that, but then I realized, the way the "advanced" API is structured, this would only work properly for the early exaggeration phase since the calls to optimize are stateless.  So the learning rate during the early exaggeration phase would be N/12 and during the normal phase just N/1 (which I guess isn't right). This could work properly only in `TSNE.fit`.  So instead I opted for this, which should be okay in most cases. I don't think many people go around changing the default early exaggeration strength.

Reply:
OK makes sense.